### ISSUE TITLE
Very difficult to dismiss the new menu while VoiceOver is active

### ISSUE THREAD

**Issue by kylehickinson** [brave/brave-ios#3838]
### Description: 

While VoiceOver is active it is very difficult/near impossible to dismiss the new menu once its been displayed.

We should remedy this in two ways:
- Support the scrub gesture (2-finger z shape swipe -- "Dismiss an alert or return to the previous screen") which will trigger menu dismissal
- Increase the space from the top of the screen to add the ability to tap the background to dismiss

### Steps to Reproduce 
  1. Turn on VoiceOver
  2. Open the new menu (•••)

**Actual result:** You can't dismiss the new menu with VoiceOver gestures

**Expected result:** You should be able to dismiss the new menu without selecting Playlist or an action that dismisses it for you such as page actions

**Reproduces how often:** Always while VoiceOver is active

**Brave Version:** 1.26+

**Comment by srirambv** [comment]
Verification passed on iPhone XR with iOS 13.5 running 1.28(21.7.8.16)
- Verified test plan from https://github.com/brave/brave-ios/pull/3845
- Verified drawing z shape with two fingers hide the menu
- Verified with voice over activated there is enough space between the menu
-----------
Verification passed on iPhone 7+ with iOS 14.5.1 running 1.28(21.7.8.16)
- Verified test plan from https://github.com/brave/brave-ios/pull/3845
- Verified drawing z shape with two fingers hide the menu
- Verified with voice over activated there is enough space between the menu
------------
Verification passed on iPad Pro with iOS 14.6 running 1.28(21.7.8.16)
- Verified test plan from https://github.com/brave/brave-ios/pull/3845
- Verified drawing z shape with two fingers hide the menu
- Verified with voice over activated there is enough space between the menu

**Comment by srirambv** [comment]
Verification passed on iPhone XR with iOS 13.5 running 1.28(21.7.8.16)
- Verified test plan from https://github.com/brave/brave-ios/pull/3845
- Verified drawing z shape with two fingers hide the menu
- Verified with voice over activated there is enough space between the menu
-----------
Verification passed on iPhone 7+ with iOS 14.5.1 running 1.28(21.7.8.16)
- Verified test plan from https://github.com/brave/brave-ios/pull/3845
- Verified drawing z shape with two fingers hide the menu
- Verified with voice over activated there is enough space between the menu
------------
Verification passed on iPad Pro with iOS 14.6 running 1.28(21.7.8.16)
- Verified test plan from https://github.com/brave/brave-ios/pull/3845
- Verified drawing z shape with two fingers hide the menu or double tap outside closes the menu
